frontend/src/components/smart/CharacterList.vue

```diff
@@ -165,15 +155,24 @@ export default {
 
       this.$emit('character-filters-changed');
     },
+
+    priceSortText(type) {
+      if (type === '1') {
+        return 'Price: Low -> High';
+      } else if (type === '-1') {
+        return 'Price: High -> Low';
+      }
+      return 'Any';
+    },
   },
 
   components: {
     CharacterArt,
   },
 
   mounted() {
-    this.levelFilter = localStorage.getItem('character-levelfilter') || '';
-    this.elementFilter = localStorage.getItem('character-elementfilter') || '';
+    this.levelFilter = sessionStorage.getItem('character-levelfilter') || '';
```

Comment:
please do not change where things are stored. they should stay in localstorage for now.

Reply:
unless it is sessionstorage everywhere else that is, but I don't think we've done that yet.